src/stories/Design System - App UI/Gdpr/index.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  import { trackGdprAccept } from '$lib/analytics/events/onboarding.js'
+  import { useCustomerCtx } from '$lib/ctx/customer/index.svelte.js'
+  import { Gdpr } from '$lib/ui/app/LoginPrompt/index.js'
+
+  const currentUser = useCustomerCtx()
+
+  function onAccept(username: string) {
+    if (currentUser.currentUser.$$) {
+      currentUser.currentUser.$$.privacyPolicyAccepted = true
+    }
+
+    if (window.onGdprAccept) window.onGdprAccept()
```

Comment:
I moved the analytics logic and policy update outside, removing side effects from the component for better readability and reusability